src/main/java/land/oras/utils/JsonUtils.java

```diff
@@ -36,6 +42,10 @@ private JsonUtils() {
         // Hide constructor
     }
 
+    public static Map<String, FileStore.Credential> fromJson(FileReader reader, Type type) {
```

Comment:
I don't think we should break add any reference of FileStore.Credential. It's generic

If you need to add a new method to read from a FileReader instead of String, use the same pattern as other method.

But don't add reference of FileStore in JsonUtils

Reply:
Thanks, changed method signature    public static <T> T fromJson(Path path, Type type) {